### PR TITLE
carousel translation, ref #329

### PR DESCRIPTION
**Pull request by silentcloud**
demo 里那段中文我去掉了，有对应 issue 链接就可以了，里面有对应解决方案；

### CODE REVIEW

**Comment by warmhug** [comment]
好的，我把这段解释移到 issue 里去了

**Comment by paranoidjk** [comment]
@silentcloud ~~transition~~ --> translation，帮你改了两次标题了哈哈

File: components/carousel/__tests__/__snapshots__/demo.test.js.snap [hunk 1]
```diff
@@ -421,7 +421,7 @@ exports[`renders ./components/carousel/demo/basic.tsx correctly 1`] = `
       allowFontScaling={true}
       ellipsizeMode="tail"
     >
-      Carousel 根据内容自适应高度
+      Carousel can be adjusted height based on content
```
Comment (paranoidjk) [inline components/carousel/__tests__/__snapshots__/demo.test.js.snap]: Carousel will adjusted height based on content

File: components/carousel/demo/basic.md [hunk 2]
```diff
@@ -5,16 +5,10 @@ title:
   en-US: Basic
 ---
 
-使用走马灯时，会遇到“每个项目“的”高度变化“时如何处理的问题：
+When using `Carousel` in web projects, you will have the problem of how to deal with each item which height will be changed.
```
Comment (paranoidjk) [inline components/carousel/demo/basic.md]: ~~you will have the problem of how to deal with each item which height will be changed.~~

you may have problem about how to deal with variable item height

File: components/carousel/demo/basic.tsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ export default class BasicCarouselExample extends React.Component<any, any> {
               <Text>Carousel 5</Text>
             </View>
           </Carousel>
-          <Text>Carousel 根据内容自适应高度</Text>
+          <Text>Carousel can be adjusted height based on content</Text>
```
Comment (paranoidjk) [inline components/carousel/demo/basic.tsx]: Carousel will adjusted height based on content

File: components/carousel/index.en-US.md [hunk 4]
```diff
@@ -4,24 +4,22 @@ type: Data Display
...
-| autoplayInterval |  Interval for autoplay iteration |    Number  |  3000         |
-| infinite | 是否循环播放 | Boolean   | false |
-| afterChange  | 切换面板后的回调函数     | (current: number): void  | 无
-| easing (`web only`) | 动画效果 | String   | linear |
-| beforeChange  (`web only`)     | 切换面板前的回调函数     | (from: number, to: number): void | 无
-| onScrollBeginDrag (`rn only`) | 见 react-native scrollView onScrollBeginDrag | (): void | 无 |
-| bounces (`rn only`) | 见 react-native scrollView bounces       |   Boolean      | true |
+| selectedIndex |  current selected index  |  number  |  0  |
+| dots | whether to display the indication dots | Boolean   | true |
+| vertical | whether is displayed vertically for content (`web`) or for pagination (`react-native`)) | Boolean   | false |
```
Comment (paranoidjk) [inline components/carousel/index.en-US.md]: for web, this props control the content display direction； for react-native, contorl the pagination display direction.

这个api怎么这么设计，web和rn的意义都不同，应该用两个API？

**Comment by silentcloud** [comment]
@paranoidjk transition 写习惯了。。。

File: components/carousel/index.en-US.md (hunk 4 above)
Comment (silentcloud) [inline components/carousel/index.en-US.md]: 你翻译的意思是 ok 的，是一个 api，只是表现形式不一样

**Pull request merged**